media/static-bundles.json

```diff
@@ -1583,15 +1583,29 @@
     {
       "files": [
         "js/base/protocol/init-details.es6.js",
+        "js/base/protocol/init-footer.es6.js",
+        "js/base/protocol/init-lang-switcher.es6.js",
+        "js/base/protocol/protocol-sub-navigation.js",
+        "js/base/base-page-init.js",
         "js/base/protocol/protocol-menu.js",
         "js/base/protocol/protocol-navigation.js",
-        "js/base/protocol/init-navigation.js",
+        "js/base/protocol/init-navigation.js"
+      ],
+      "name": "ui"
+    },
+    {
+      "files": [
+        "js/base/protocol/init-details.es6.js",
         "js/base/protocol/init-footer.es6.js",
```

Comment:
**Question/suggestion:** is `"js/base/protocol/init-footer.es6.js"` needed for the redesigned footer still? If not, can we remove this from `m24-ui`?